docs/Makefile

```diff
@@ -10,6 +10,7 @@ include docs.mk
 PODMAN := $(shell if command -v podman >/dev/null 2>&1; then echo podman; else echo docker; fi)
 BUILD_IN_CONTAINER ?= true
 
+.PHONY: sources/setup/install/helm/reference.md
```

Comment:
Otherwise when running `make -C docs sources/setup/install/helm/reference.md` like the CI jobs recommended, my `make` didn't think there was anything to update.. is there a flag I'm missing you'd prefer me use rather than `.PHONY` ?

Reply:
I'm confused why this needs to be a `.PHONY` when this file does exist. maybe you need to `clean` first? But I'm guessing it failed because the file existed even though the contents weren't what they should be, and the `.PHONY` just forces it to always run? I guess that's a fair trade off vs having to remember to `clean` everytime.

Reply:
Hey yeah sorry `clean` should have been obvious to me, but I guess I am use to codegen/docgen/chartgen/etc. tooling via Makefile (like kubebuilder's make targets for example) always running / re-generating code since it should be deterministic, and therefore you don't have to remember to run `clean` which is nice.

I recognize this is a bit overstepping in the PR's scope and could have unintended consequences in the CI workflows,etc. potentially so if you'd prefer I back this out that's fine with me!